### ISSUE TITLE
'Derived from' could be 'Remixed from'

### ISSUE THREAD

**Issue by rmillwood** [backface/turtlestitch#78]
By making 'Derived from' 'Remixed from', we tap into Scratcher's knowledge instead of introducing a new term!

**Comment by backface** [comment]
good point! The reasoning why I called it "derived from" is to have a bit more optical difference between the two headlines, which otherwise would be "remixed from" and "remixed by". But I do not have a hard opinion here.

**Comment by backface** [comment]
any more opinions on that?

**Comment by rmillwood** [comment]
I understand your reasoning, but I think it is better to tap in to the Scratch culture and terminology by using the same word used there in both cases. Actually you made me look again at Scratch 3, and I discover they removed the link to the /remixtree and instead show 'thanks' to each of the predecessors! Then at the bottom right you can see 'Remixes'. Nothing ever stays still!

**Comment by UrticaDesign** [comment]
I agree - use the same term as in the Scratch environment. I like the term remixing. It makes it so legal to do - if you understand... ;-)
